docs/platform/concepts/service_backups.rst

```diff
@@ -6,14 +6,18 @@ This article provides information on general rules for handling service backups
 About backups at Aiven
 ----------------------
 
-All Aiven services, except for Apache Kafka® and M3 Aggregator/Coordinator, have time-based backups that are encrypted and securely stored. The backup retention times vary based on the service and the selected service plan. Backups we take for managing the service are not available for download for any service type as they are compressed and encrypted by our management platform.
+All Aiven services, except for Apache Kafka® and M3 Aggregator/Coordinator, have time-based backups that are encrypted and securely stored. The backup retention times vary based on the service and the selected service plan. 
+
+Backups taken by Aiven for managing the service are not available for download for any service type. This is because they are compressed and encrypted by our management platform.
```

Comment:
Consider: I'd make it clear that these backups are something different than the ones above.

Aiven also takes service backups for managing purposes. These backups are compressed and encrypted by the Aiven management platform and, as such, not available for download for any service type.